### PR TITLE
feats(transformers)add mobilevit,mobilevitv2,deepseek_v3,xlnet models

### PR DESCRIPTION
**Pull request by iugoood**
# Add
1 add mobilevit,mobilevitv2,deepseek_v3,xlnet models
2 add UT
# Usage
## mobilevit
        >>> import requests
        >>> import torch
        >>> from PIL import Image
        >>> from transformers import AutoImageProcessor
        >>> from mindone.transformers import MobileViTForSemanticSegmentation
        >>> import numpy as np
        >>> import mindspore as ms
        >>> url = "http://images.cocodataset.org/val2017/000000039769.jpg"
        >>> image = Image.open(requests.get(url, stream=True).raw)
        >>> image_processor = AutoImageProcessor.from_pretrained("apple/deeplabv3-mobilevit-small")
        >>> model = MobileViTForSemanticSegmentation.from_pretrained("apple/deeplabv3-mobilevit-small")
        >>> inputs = image_processor(images=image, return_tensors="np")
        >>> for key,value in inputs.items():
        >>>     if isinstance(value, np.ndarray):
        >>>         inputs[key] = ms.tensor(value)
        >>>     elif isinstance(value, list):
        >>>         inputs[key] = ms.tensor(value)
        >>>     if key == "pixel_values":
        >>>         inputs[key] = inputs[key].to(ms.float32)
        >>> outputs = model(**inputs)
        >>> # logits are of shape (batch_size, num_labels, height, width)
        >>> logits = outputs.logits
 ## mobilevitv2
        >>> import requests
        >>> import torch
        >>> from PIL import Image
        >>> from transformers import AutoImageProcessor
        >>> from mindone.transformers import MobileViTV2ForSemanticSegmentation
        >>> import numpy as np
        >>> import mindspore as ms
        >>> url = "http://images.cocodataset.org/val2017/000000039769.jpg"
        >>> image = Image.open(requests.get(url, stream=True).raw)
        >>> image_processor = AutoImageProcessor.from_pretrained("apple/mobilevitv2-1.0-imagenet1k-256")
        >>> model = MobileViTV2ForSemanticSegmentation.from_pretrained("apple/mobilevitv2-1.0-imagenet1k-256")
        >>> inputs = image_processor(images=image, return_tensors="np")
        >>> for key,value in inputs.items():
        >>>     if isinstance(value, np.ndarray):
        >>>         inputs[key] = ms.tensor(value)
        >>>     elif isinstance(value, list):
        >>>         inputs[key] = ms.tensor(value)
        >>>     if key == "pixel_values":
        >>>         inputs[key] = inputs[key].to(ms.float32)
        >>> outputs = model(**inputs)
        >>> # logits are of shape (batch_size, num_labels, height, width)
        >>> logits = outputs.logits
## deepseek_v3
        >>> from transformers import AutoTokenizer
        >>> from mindone.transformers import DeepseekV3ForCausalLM
        >>> import mindspore as ms
        >>> model = DeepseekV3ForCausalLM.from_pretrained("meta-deepseek_v3/DeepseekV3-2-7b-hf")
        >>> tokenizer = AutoTokenizer.from_pretrained("meta-deepseek_v3/DeepseekV3-2-7b-hf")
        >>> prompt = "Hey, are you conscious? Can you talk to me?"
        >>> inputs = tokenizer(prompt, return_tensors="np")
        >>> # Generate
        >>> generate_ids = model.generate(ms.tensor(inputs.input_ids), max_length=30)
        >>> tokenizer.batch_decode(generate_ids, skip_special_tokens=True, clean_up_tokenization_spaces=False)[0]
## xlnet
        >>> from transformers import AutoTokenizer
        >>> from mindone.transformers import XLNetLMHeadModel
        >>> import mindspore as ms
        >>> tokenizer = AutoTokenizer.from_pretrained("xlnet/xlnet-large-cased")
        >>> model = XLNetLMHeadModel.from_pretrained("xlnet/xlnet-large-cased")
        >>> # We show how to setup inputs to predict a next token using a bi-directional context.
        >>> input_ids = ms.tensor(
        ...     tokenizer.encode("Hello, my dog is very <mask>", add_special_tokens=False)
        ... ).unsqueeze(
        ...     0
        ... )  # We will predict the masked token
        >>> perm_mask = mint.zeros((1, input_ids.shape[1], input_ids.shape[1]), dtype=ms.float32)
        >>> perm_mask[:, :, -1] = 1.0  # Previous tokens don't see last token
        >>> target_mapping = mint.zeros(
        ...     (1, 1, input_ids.shape[1]), dtype=ms.float32
        ... )  # Shape [1, 1, seq_length] => let's predict one token
        >>> target_mapping[
        ...     0, 0, -1
        ... ] = 1.0  # Our first (and only) prediction will be the last token of the sequence (the masked token)
        >>> outputs = model(input_ids, perm_mask=perm_mask, target_mapping=target_mapping)
        >>> next_token_logits = outputs[
        ...     0
        ... ]  # Output has shape [target_mapping.size(0), target_mapping.size(1), config.vocab_size]
        >>> # The same way can the XLNetLMHeadModel be used to be trained by standard auto-regressive language modeling.
        >>> input_ids = ms.tensor(
        ...     tokenizer.encode("Hello, my dog is very <mask>", add_special_tokens=False)
        ... ).unsqueeze(
        ...     0
        ... )  # We will predict the masked token
        >>> labels = ms.tensor(tokenizer.encode("cute", add_special_tokens=False)).unsqueeze(0)
        >>> assert labels.shape[0] == 1, "only one word will be predicted"
        >>> perm_mask = mint.zeros((1, input_ids.shape[1], input_ids.shape[1]), dtype=ms.float32)
        >>> perm_mask[
        ...     :, :, -1
        ... ] = 1.0  # Previous tokens don't see last token as is done in standard auto-regressive lm training
        >>> target_mapping = mint.zeros(
        ...     (1, 1, input_ids.shape[1]), dtype=ms.float32
        ... )  # Shape [1, 1, seq_length] => let's predict one token
        >>> target_mapping[
        ...     0, 0, -1
        ... ] = 1.0  # Our first (and only) prediction will be the last token of the sequence (the masked token)
        >>> outputs = model(input_ids, perm_mask=perm_mask, target_mapping=target_mapping, labels=labels)
        >>> loss = outputs.loss
        >>> next_token_logits = (
        ...     outputs.logits
        ... )  # Logits have shape [target_mapping.size(0), target_mapping.size(1), config.vocab_size]
# Performance
Experiments are tested on Ascend Atlas 800T A2 machines with mindspore 2.6.0.
|model | mode |speed|
|---|---|---|
|mobilevit| pynative| 32.63 token/s|
|mobilevitv2|pynative|45.82 token/s|
|deepseek_v3|pynative|3.32 token/s|
|xlnet|pynative|2.71 token/s|

### CODE REVIEW

**Comment by Fzilan** [comment]
could you please fix your sample usages?
